content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -91,73 +103,89 @@ For example:
      "type": "MODIFIED",
      "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "11020", ...}, ...}
    }
-   ...
+   …
    ```
 
-A given Kubernetes server will only preserve a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default.  When the requested watch operations fail because the historical version of that resource is not available, clients must handle the case by recognizing the status code `410 Gone`, clearing their local cache, performing a list operation, and starting the watch from the `resourceVersion` returned by that new list operation. Most client libraries offer some form of standard tool for this logic. (In Go this is called a `Reflector` and is located in the `k8s.io/client-go/cache` package.)
+   {{< note >}}
+   Although the API server responded with a `Content-Type` header indicating a JSON response,
+   the response to a _watch_ request is in fact a stream of multiple documents. For JSON format, the client can
+   detect the end of each JSON document and begin parting the next one.
+   {{< /note >}}
+
+A given Kubernetes cluster only retains a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default. If a requested Watch operation fails because the historical version of that resource is not available, clients must handle the case by recognizing the HTTP response status 410 (Gone). When a client sees a 410 response it should clear any relevant records from its local cache, perform a new List operation, and request a new Watch that starts from the `resourceVersion` returned by that new List operation.
+
+Most Kubernetes client libraries offer some form of standard tool for this Watch retry logic. (In Go this is called a `Reflector` and is located in the `k8s.io/client-go/cache` package.)
 
 ### Watch bookmarks
 
-To mitigate the impact of short history window, we introduced a concept of `bookmark` watch event. It is a special kind of event to mark that all changes up to a given `resourceVersion` the client is requesting have already been sent. Object returned in that event is of the type requested by the request, but only `resourceVersion` field is set, e.g.:
+To mitigate the impact of short history window, Kubernetes provides `bookmark` Watch events. A _bookmark_ is a special kind of event to mark that all changes up to a given `resourceVersion` the client is requesting have already been sent. The object returned in that event is of the type requested by the request, but only `resourceVersion` field is set.
```

Comment:
nit: To mitigate the impact of a short history window,

Reply:
Possible edit?
A _bookmark_ is a special kind of event that marks all changes occurring before a client requests a given `resourceVersion` as sent by the API server. The object returned in the bookmark event is the same as the type set in the client request, but only the `resourceVersion` field is set.

Reply:
IIUC, the bookmark watch event should be spelled as `BOOKMARK`. As shown in the example on line 139.

Reply:
Addressed, I think.